.github/workflows/pull-request-status-check.yml

```
@@ -32,6 +32,13 @@ jobs:
       - name: Check the format of the json/jsonc files
         run: sh ./.github/scripts/check-json-format.sh
 
+  check_include_style:
+    name: Check include directive style ("" vs <>)
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)